cypress/plugins/index.js

```diff
@@ -11,7 +11,21 @@
 // This function is called when a project is opened or re-opened (e.g. due to
 // the project's config changing)
 
-module.exports = function() {
-  // `on` is used to hook into various events Cypress emits
-  // `config` is the resolved Cypress config
+const mongoose = require('mongoose');
+
+// the below 3 lines are commented out to test the mongoose drop function.
+// module.exports = function() {
+//   // `on` is used to hook into various events Cypress emits
+//   // `config` is the resolved Cypress config
+// }
+
+module.exports = async (on, config) => {
+  on('task', {
+    async emptyPosts() {
+      // this is a new task that cypress understands, to delete the posts from the db
+      var con = mongoose.connect("mongodb://0.0.0.0/acebook_test");
```

Comment:
Probably the variable con might not be necessary? But as it is working, we can keep it here for now and remove it in later branches.